### PR TITLE
add 1.0 weight for </s>

### PR DESCRIPTION
**Pull request by juliakreutzer**
</s> previously had weight 0, which made a difference between standard xent training and feedback with only 1s. 
Also directly fixing the bug found by @heidelkin with the missing .cuda() for weights.

### CODE REVIEW

**Comment by juliakreutzer** [comment]
@heidelkin please see if that fixes the issue

**Review by heidelkin — COMMENTED** [review]
After the fix, the training curve and validation performance of a token level reward system with all tokens receiving a reward of 1 is ALMOST identical, e.g., up to two or three decimal points in BLEU, to a NMT trained by xent loss given the same pre-trained model. 

However, the slight-difference issue above remains for two systems trained using standard xent loss given identical settings. I am not sure where the stochasticity comes from.

**Comment by juliakreutzer** [comment]
Good. I understand this as an approval of the code ;)
There is still some randomness caused by CUDA functions, as explained [here](https://pytorch.org/docs/stable/notes/randomness.html).
